<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmMain</class>
 <widget class="QMainWindow" name="FrmMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FrmMain</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_10">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QGroupBox" name="grpBoxEintragen">
          <property name="title">
           <string>Klausur Hinzufügen</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Fach eingeben</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="edtFach"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Note eingeben</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="edtNotenEingeben"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Datum eingeben</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLineEdit" name="edtDatum"/>
               </item>
               <item>
                <widget class="QToolButton" name="btnHoleDatum">
                 <property name="text">
                  <string>Auswählen</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="btnklausurEintragen">
             <property name="text">
              <string>Klausur eintragen</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="grpBoxAenderungLoeschen">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="title">
           <string>Änderung/Löschen</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLineEdit" name="edtNoteEintragen"/>
             </item>
             <item>
              <widget class="QPushButton" name="btnNoteEintragen">
               <property name="text">
                <string>Note eintragen</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="btnEintragLoeschen">
             <property name="text">
              <string>Klausur löschen</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnDurchschnittZumFach">
             <property name="text">
              <string>Durchschnittsnote zum Fach berechnen</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnAuswahlAufheben">
             <property name="text">
              <string>Auswahl aufheben</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Klausuren anzeigen</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="lwAnzeige"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Suchtext eingeben</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLineEdit" name="edtSuchtext"/>
              </item>
              <item>
               <widget class="QPushButton" name="btnSuchen">
                <property name="text">
                 <string>Suchen</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Suchtext anzeigen</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="lwSuchtextAnzeige"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>610</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
